src/components/NcContent/NcContent.vue

```diff
@@ -72,18 +74,68 @@ It includes the Navigation, the App content and the Sidebar.
 	<div id="content-vue"
 		:class="`app-${appName.toLowerCase()}`"
 		class="content">
+		<Teleport to="#skip-actions">
+			<NcButton type="primary" href="#app-content-vue">
+				{{ t('Skip to main content') }}
+			</NcButton>
+			<NcButton v-show="hasAppNavigation"
+				type="primary"
+				href="#app-navigation-vue"
+				@click.prevent="openAppNavigation">
+				{{ t('Skip to app navigation') }}
+			</NcButton>
+		</Teleport>
 		<slot />
 	</div>
 </template>
 
 <script>
+import { emit } from '@nextcloud/event-bus'
+import { t } from '../../l10n.js'
+
+import NcButton from '../NcButton/NcButton.vue'
+// TODO: This is built-in for vue3 just drop the import
+import Teleport from 'vue2-teleport'
```

Comment:
I'm a bit worried about this dependency. It has just 8 commits when it was created and doesn't seem to be actively used or maintained.

There is `vue-simple-portal` from the author of famous `vue-portal` library and the initiator of [adding a Teleport component to Vue](https://www.github.com/vuejs/rfcs/pull/112).

It has a little bit different interface (prop is called `selector` instead of `to`), but it is used by thousands of other packages and looks more reliable. 

https://github.com/LinusBorg/vue-simple-portal

Reply:
Both packages have nearly the same amount of weekly downloads and both have their last real update 3 years ago.

Reply:
Weekly download stats are shown only for the last year, when Vue 3 was released already for quite a long time. I guess most of the users of `vue2-teleport` searched for "Vue 2 teleport" as an alternative from Vue 3. But even if compare only weekly downloads last year, Linus' are a little bit more.

The full version (`portal-vue`) still has almost half a million downloads per week in comparison with 10-13k.

I'd consider a library from a Vue team member and author of well-known `portal-vue` less risky.

Reply:
I am fine with both, for `next` it is then just one line more to adjust. I will change this.

Reply:
I adjusted it but this will require a bit more work as that library injects a wrapper around the teleported content, thus the server styling is broken.